src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -413,6 +413,12 @@ protected function getRelationshipFromMethod($method)
         $relation = $this->$method();
 
         if (! $relation instanceof Relation) {
+            if (is_null($relation)) {
+                throw new LogicException(sprintf(
+                    '%s::%s must return a relationship instance, but "null" was returned. Was the "return" keyword used?', static::class, $method
```

Comment:
Suggestion:
- ~Was the "return" keyword used?~
- Is the "return" keyword missing?